inference-engine/src/mkldnn_plugin/utils/node_dumper.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#ifdef CPU_DEBUG_CAPS
+
+#include "node_dumper.h"
+
+#include "mkldnn_node.h"
+#include "utils/blob_dump.h"
+
+#include "ie_common.h"
+#include <array>
+#include <regex>
+#include <sstream>
+#include <string>
+
+using namespace InferenceEngine;
+
+namespace MKLDNNPlugin {
+
+void NodeDumper::setup() {
```

Comment:
Why do we need separate setup function()? Cannot we just implement the same inside ctor?

Reply:
For readability and to keep constructor code clean.
Just a habit, I see no problems to remove it.

Reply:
Done